src/pybind/mgr/cephadm/serve.py

```diff
@@ -311,6 +319,16 @@ def refresh(host: str) -> None:
                 if r:
                     bad_hosts.append(r)
 
+            if self.mgr.cache.host_needs_custom_logrotate_file(host):
+                for logrotate_type in [LogrotateConfigType.cephadm, LogrotateConfigType.cluster]:
```

Comment:
Since you're iterating over all the values of LogrotateConfigType you can just iterate over LogrotateConfigType.
```
for x in LogrotateConfigType:
   print(x)
```